src/Avatar.svelte

```diff
@@ -1,14 +1,19 @@
 <script>
   import i from 'initials';
+  import {getRandomColor} from './utils.js';
 
   export let style = '';
   export let name = 'Avatar';
   export let initials;
   export let src;
-  export let bgColor = 'lightGray';
+  export let bgColor;
   export let textColor = 'white';
   export let size = '50px';
   export let borderRadius = '50%';
+  export let square = false;
+  export let colored = false;
+
+  const background = bgColor ? bgColor : colored ? getRandomColor() : 'lightGrey';
```

Comment:
We can simplify this using the default value in the `bgColor` declaration, and for control the background we can just use a simple ternary like this:

`const background = colored ? getRandomColor() : bgColor;`

Reply:
good idea, thanks i'll revise it 👍